fuzzers/074-dump_all/reduce_site_types.py

```diff
@@ -0,0 +1,55 @@
+""" Reduce sites types to prototypes that are always correct.
```

Comment:
what is a prototype?

Reply:
I'm presuming a generic description of something as opposed to a specific instance

Reply:
Yes, exactly.